FastAutoAugment/hardness_measures.py

```diff
@@ -0,0 +1,64 @@
+import torch
+import numpy as np
+from collections import defaultdict
+from torch.nn import CosineSimilarity, CrossEntropyLoss
+from FastAutoAugment.factory import Factory
+
+class AVH:
+    def __init__(self, update_method: str = "no_change", classwise_normalization: bool = False):
+        self.update_method = update_method
+        self.classwise_normalization = classwise_normalization
+        
+    def __call__(self, model, embeddings, targets) -> list:
+        """Calculate AVH scores per instance in a batch
+        :param network: network object of the experiment
+        :type network: network object
+        :param embeddings: embeddings with size 
+            [batch_size, #nodes in last but one layer of the neural network]
+        :type embeddings: Any
+        :param targets: Targets (Ground Truth)
+        :type targets: Any
+        :return: List of AVH values per instance
+        """
+#         import ipdb; ipdb.set_trace();
+        weights_list = list(model.state_dict().keys())
+        weight_matrix = model.state_dict()[weights_list[-2]]
```

Comment:
do check if this is getting the right weight value

Reply:
I had checked the shape of the weight_matrix. It was correct.